src/components/calculator/SelectControl.tsx

```diff
@@ -25,12 +25,36 @@ function showRiskMultiplier(multiplier: number): string {
   }
 }
 
+interface LabelFormat {
+  label: string
+  header?: string
+  subLabel?: string
+}
+
+export const labelMaker = function ({
```

Comment:
Maybe just make this a React Component? (see e.g. signature for GenericSelectControl).

Also, maybe call it something more descriptive, like LabelWithHeader?

Alternatively, this behavior could put put into GenericSelectControl.

Reply:
(done)